skflow/data_feeder.py

```diff
@@ -176,3 +175,76 @@ def _feed_dict_fn():
                     out[i] = y
             return {input_placeholder.name: inp, output_placeholder.name: out}
         return _feed_dict_fn
+
+
+class DaskDataFeeder(object):
+    """Data feeder for TF trainer that reads data from dask.Series.
+
+    Numpy arrays can be serialized to disk and it's possible to do random seeks into them.
+    DaskDataFeeder will remove requirement to have full dataset in the memory and still do
+    random seeks for sampling of batches.
+
+    Parameters:
+        X: iterator that returns for each element, returns features.
+        y: iterator that returns for each element, returns 1 or many classes /
+           regression values.
+        n_classes: indicator of how many classes the target has.
+        batch_size: Mini batch size to accumulate.
+
+    Attributes:
+        X: input features.
+        y: input target.
+        n_classes: number of classes.
+        batch_size: mini batch size to accumulate.
+        input_shape: shape of the input.
+        output_shape: shape of the output.
+        input_dtype: dtype of input.
+        output_dtype: dtype of output.
+    """
+    def __init__(self, X, y, n_classes, batch_size, random_state=None):
+        import dask.dataframe as dd
+        x_dtype = np.int64 if X.dtype == np.int64 else np.float32
+        self.X = check_array(X, ensure_2d=False,
+                             allow_nd=True, dtype=x_dtype)
+        self.y = check_array(y, ensure_2d=False, dtype=np.float32)
+        self.X = X
```

Comment:
hm... this is weird, why first you do `self.X = check_array(...)` and then `self.X = X`?